### PR TITLE
add Factory for Algolia Place Provider

### PR DESCRIPTION
**Pull request by Fabex**
Since my pull request on geocoder-php/Geocoder https://github.com/geocoder-php/Geocoder/pull/854
is merged and released, I forgot to make the pull request on BazingaGeocoderBundle.

So I add the Factory for Algolia Place Provider

### CODE REVIEW

**Comment by tgalopin** [comment]
I would be glad to have this available :) !

File: ProviderFactory/AlgoliaPlacesFactory.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    protected static function configureOptionResolver(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'httplug_client' => null,
+            'apiKey' => null,
+            'appId' => null,
+        ]);
+
+        $resolver->setAllowedTypes('httplug_client', ['object', 'null']);
+        $resolver->setAllowedTypes('apiKey', ['string']);
```
Comment (tgalopin) [inline ProviderFactory/AlgoliaPlacesFactory.php]: What about using `api_key` and `app_id` instead to be consistent with the others factories?

Comment (Nyholm) [reply ProviderFactory/AlgoliaPlacesFactory.php]: 👍

**Comment by norkunas** [comment]
@Fabex are you willing to finish this?

**Comment by norkunas** [comment]
This was implemented in #247.

**Pull request closed without merging**